client/extensions/wp-super-cache/index.js

```diff
@@ -12,6 +12,6 @@ import { settings } from './controller';
 import { Tabs } from './constants';
 
 export default function() {
-	page( '/extensions/wp-super-cache', siteSelection, sites, navigation, settings );
+	page( '/extensions/wp-super-cache', sites, navigation, settings );
```

Comment:
That's a bit of a misnomer -- the `siteSelection` middleware looks at the current URL and (somewhat heuristically) determines the currently selected site from it. For the all-sites route, that doesn't really make any sense since there is no site fragment in the URL. Instead, we use the `sites` middleware, which is that sites list the user is prompted with to select a site from (and which subsequently redirects to a URL that includes that site's domain).

Now that I think of it, this also means we can drop the `settings` middleware from the all-sites route; and probably even `navigation`, which is the sidebar (which isn't present here). I'll give that a try...